src/components/ArticleComments.jsx

```diff
@@ -0,0 +1,149 @@
+                            body: comment,
+                            votes: 0,
+                        };
+                        return [contextRender, ...currComments];
+                    });
+                })
+                .catch((err) => {
+                    return setPostCommentErrStatus(err);
+                });
+        }
+    }
+
+    function notificationHandler() {
+        setNotificationStatus(false);
+    }
+
+    function postErrHandler() {
+        setPostCommentErrStatus(null);
+    }
+
+    function postLengthErrHandler() {
+        setPostLengthErrStatus(null);
+    }
+
+    if (postLengthErrStatus) {
+        return (
+            <ErrorPopup
+                errStatus={"comment must be between 3 and 200 characters"}
+                onClose={postLengthErrHandler}
+            />
+        );
+    }
+
+    if (notificationStatus) {
+        return (
+            <NotificationPopup
+                status={"comment posted"}
+                onClose={notificationHandler}
+            />
+        );
+    }
+
+    if (commentsStatus) {
+        return <p>Fetching comments</p>;
+    }
+
+    if (postCommentErrStatus) {
+        return (
+            <ErrorPopup
+                errStatus={"unable to submit comment, try again soon"}
+                onClose={postErrHandler}
+            />
+        );
+    } else {
+        return (
+            <>
+                <section className="post_comment_container">
+                    <form className="post_comment_form" onSubmit={handleSubmit}>
+                        <div className="post_comment_labels_container">
+                            <label htmlFor="body">
+                                Post comment:<br></br>
+                                <input
+                                    name="body"
+                                    type="text"
+                                    maxLength={200}
+                                    value={comment.body}
+                                    onChange={handleChange}
+                                />
+                            </label>
+                            <button
+                                className="post_comment_submit_button"
+                                onClick={handleSubmit}
+                            >
+                                Submit
+                            </button>
+                        </div>
+                    </form>
+                </section>
+                <section className="article_comments_list_container"></section>
+                <ul>
+                    {comments.map((commentsMapped) => {
+                        return (
+                            <li type="none" key={commentsMapped.comment_id}>
+                                <CommentCard comment={commentsMapped} />
+                            </li>
+                        );
+                    })}
+                </ul>
+            </>
+        );
+    }
+}
```

Comment:
Currently while the comment is posting, the user is able to mash the post button and post multiple of the same comments before the pop up appears. Could you prevent the user from posting a comment while one is currently being posted? You could achieve this by temporarily disabling the post button ⏹️